loadgen/kitchen_sink_executor_test.go

```diff
@@ -152,8 +152,9 @@ func TestKitchenSink(t *testing.T) {
 						}),
 				},
 			},
+			// Note: workflowId is now "my-child-{parent-workflow-id}" due to concurrent execution support
 			historyMatcher: PartialHistoryMatcher(`
-				StartChildWorkflowExecutionInitiated {"workflowId":"my-child"}`),
+				StartChildWorkflowExecutionInitiated`),
 		},
 		{
```

Comment:
Would it be reasonable to add a test for the new actions? Without it, only the Go implementation is tested right now.